docs/blog/table-scroll-ghost.en-US.md

```
@@ -0,0 +1,187 @@
+title: Table Scroll Goes Off Track
+date: 2026-07-08
+author: zombieJ
+---
+
+Recently we ran into a [Table horizontal scroll misalignment issue](https://github.com/ant-design/ant-design/issues/57850). In a virtual scrolling scenario, the user dragged the horizontal scrollbar and the table body had already reached the correct position, but the header would occasionally stay at an old position. This was especially obvious when scrolling back to the far left: the body was already at `0`, while the header still stayed somewhere in the middle.
+
+![Table horizontal scroll misalignment](https://mdn.alipayobjects.com/huamei_ktaqcm/afts/img/A*SkGeTa6hbf8AAAAARnAAAAgAeuN6AQ/original)
+
+From the symptom, it looked like several scroll areas inside Table were not synchronized correctly. Since Table has scroll linkage among the fixed header, fixed columns, summary row, and sticky scrollbar, this kind of issue can be a little winding to investigate. This article records the debugging process.
+
+### Start With Scroll Sync
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)